### PR TITLE
Updated Docs for pack application

### PR DESCRIPTION
**Pull request by saurabha994**
### Why:

Updated docs to reflect change in Github Education pack application for the role of students and teachers.

### What's being changed (if available, include any code snippets, screenshots, or gifs):

Made change to add an additional step of selecting a role while filling the application.

### Check off the following:

- [X] I have reviewed my changes in staging (look for the "Automatically generated comment" and click the links in the "Preview" column to view your latest changes).
- [X] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### Writer impact (This section is for GitHub staff members only):

- [ ] This pull request impacts the contribution experience
  - [ ] I have added the 'writer impact' label
  - [ ] I have added a description and/or a video demo of the changes below (e.g. a "before and after video")

### CODE REVIEW

**Comment by cmwilson21** [comment]
👋 @saurabha994 - Thanks so much for opening a PR! I'll get this triaged for review :zap:

File: content/education/explore-the-benefits-of-teaching-and-learning-with-github-education/use-github-in-your-classroom-and-research/apply-for-an-educator-or-researcher-discount.md [hunk 1]
```diff
@@ -26,6 +26,8 @@ For more information about user accounts on {% data variables.product.product_na
 
 {% data reusables.education.benefits-page %}
 {% data reusables.education.click-get-teacher-benefits %}
+3. Under "Which best describes your academic status?", select **Teacher**.
+  ![Select academic status](/assets/images/help/education/academic-status-teacher.png)
```
Comment (mattpollard) [inline content/education/explore-the-benefits-of-teaching-and-learning-with-github-education/use-github-in-your-classroom-and-research/apply-for-an-educator-or-researcher-discount.md:30]:
```suggestion
1. Under "Which best describes your academic status?", select **Teacher**.

   ![Select academic status](/assets/images/help/education/academic-status-teacher.png)
```

**Review by mattpollard — DISMISSED** [review]
Thanks, @saurabha994! ✨ I'll address the merge conflicts, incorporate a couple of small changes, and merge this for you.

**Edit**: Update [below](https://github.com/github/docs/pull/20955#issuecomment-1265939353) 👇🏻

**Comment by mattpollard** [comment]
👋🏻 @saurabha994, sorry for the delay. I was having a bit of trouble pushing to your branch.

Unfortunately, the file you modified no longer exists. I'm thinking perhaps `main` in your local clone of `github/docs` was a couple of months old.

The content you need to modify is now in [this file](https://github.com/github/docs/blob/main/content/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-teachers/apply-to-github-global-campus-as-a-teacher.md), which you can see on the `main` branch. You’ll need to merge `main` into your branch, and apply your change from `apply-for-an-educator-or-researcher-discount.md` to `github-global-campus-for-teachers/apply-to-github-global-campus-as-a-teacher.md` instead when you go to resolve the merge conflict.

Let me know if you have questions! 🙏🏻

**Comment by saurabha994** [comment]
Closing this in favour of another PR.

**Pull request closed without merging**